docs/robots/husky/troubleshooting_husky.mdx

```diff
@@ -1,10 +1,205 @@
+For the button to turn on:
+
+1.  The Husky needs to have a charged battery installed.
+2.  The main fuse on the Husky's MCU needs to be functioning.
+3.  The ribbon cable between the MCU and switch PCBA needs to be fully seated on both ends.
+4.  The connector on the switch's cable needs to be fully seated on the switch's PCBA.
+
+We have not seen these connections fail, even in applications with lots of vibration.
+The most common fix for this issue is to check that battery is charged, and that it's power cable is seated in the Husky's MCU.
+
+### Battery indicator is flashing
+
+Battery voltage is too low for Husky to drive the motors.
+Charge the battery and try again.
+
+### Battery indicator only shows 50% charge with a fresh battery
+
+The battery indicator is controlled by the Husky MCU's microcontroller, based on a voltage measurement of the battery.
+The controls program was designed for a lithium ion battery.
+_24 V lithium ion batteries_ reach a peak voltage of 29 V.
+Our Standard _24 V lead acid battery_ does not reach as high of a voltage, so the MCU's microcontroller thinks the battery is not fully charged.
+
+There is not a way to change this behaviour on your Husky.
+
+---
+
+## Mobility
+
+### Husky experiences momentary drops in communication
+
+1.  It's possible the robot has reached the thermal limit, or current limit.
+2.  The maximum current limits are set to 18.3 A, however, as the motors are only rated for 8A continuous, anything above that will trigger the current limit eventually, the higher above this limit, the more quickly it will trigger.
+    If you are unsure whether this is the case, feel free to contact support@clearpathrobotics.com with a rosbag of your system.
+    Clearpath robotics also offers a high torque upgrade if more torque is required for your project.
+
+### Husky's velocity is inconsistent, with a jumping motion
+
+It's possible the robot has reached the thermal limit, or current limit.
+The maximum current limits are set to 18.3 A, however, as the motors are only rated for 8 A continuous, anything above that will trigger the current limit.
+The limit will be triggered if your application sends more than 8 A of current to the motors continuously.
+
+:::note
+
+Clearpath Robotics also offers a high torque motor upgrade if more torque is required for your project.
+
+:::
+
+### Husky rolls on a slope
+
+Husky's wheels are always connected to the motors.
+When you send a motion command with `/cmd-vel`, the Husky's MCU subscribes to that message, and relays a CAN command to the Husky's motor controllers.
+These motor controllers change the voltage across the DC brushed motors, causing a mechamical torque at the wheel.
+
+When you stop publishing to `/cmd_vel`, ROS starts publishing a default value to `/cmd_vel` or zero velocity.
+The motor controllers will attempt to achieve zero velocity by monitoring the motor encoders, and increasing voltage to the motor till the encoder stops moving.
```

Comment:
Are we certain that this statement is correct? Can you confirm with @tonybaltovski? The reason I ask is that we have recently quoted a work package to implement a position controller (rather than velocity) that will hold the robot steady on a hill. Maybe I'm confusing this with a different robot?

Reply:
You are correct. The existing 2013 - 2022 Husky does not provide holding torque on a hill. The motors do provide some resistive torque, but are not trying to hold position.

I have added a new repository with firmware updating instructions per a discussion with @tonybaltovski , and detailed this in _cpr-documentation_.